scripts/markdown_diff_lint.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+# This script runs markdownlint-cli2 on changed files.
+# Usage: ./markdown_lint.sh
+
+set -eo pipefail
+
+if ! command markdownlint-cli2 dummy.md &>/dev/null; then
+  echo "markdownlint-cli2 needs to be installed."
+  echo "Install it by running npm install -g markdownlint-cli2"
```

Comment:
Could we instead just link the the markdown-cli2 install docs: https://github.com/DavidAnson/markdownlint-cli2?tab=readme-ov-file#install as personally I found the `brew install` option much more convenient for my device.

Reply:
Makes sense. Updated!